web/app/help/operations/place-vertices-with-edge-lengths.asciidoc

```diff
@@ -0,0 +1,48 @@
+### Place vertices with edge lengths
+
+These methods create a graph layout as a new ``Vector[number]`` vertex attribute
+where the edges have given lengths.
```

Comment:
Shouldn't this be worded a bit weaker? ... in a way which makes the physical distance between connected vertices be as close as possible to given edge length? Or something better. :) I mean, this sounds like we find a perfect solution.

Reply:
> Shouldn't this be worded a bit weaker?

Done, thanks.